### PR TITLE
Adds ledger instructions

### PR DESCRIPTION
**Pull request by nkcr**

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -100,7 +100,120 @@ in their associated documentation (chapter "Components").
...
-...run conodes, make ledger, get bc file, set BC env variable in variables.sh, etc.
+#### Run the nodes
+
+Build the conode binary and use the `run_nodes.sh` script in order to get a set
+of conodes up and running:
+
+```bash
+cd ledger/conode
+go build
+./run_nodes.sh -d tmp -v 2
```
Comment (jeffallen) [inline docs/setup.md]: I'm not sure what the point of `tmp` is. I used no -d arg, but if you use, it, please make it something that says, "I'm important, don't delete me", like `conode_data`.

File: docs/setup.md [hunk 2]
```diff
@@ -100,7 +100,120 @@ in their associated documentation (chapter "Components").
...
+
+Make sure to run `varialbes.sh` if you set a custom `BC_CONFIG` variables. If
+set, this variable tells `bcadmin` where to store and get the configuration
+files.
+
+Create a new skipchain:
+
+```bash
+# This file was created by the run_nodes.sh script
+bcadmin create ledger/conode/tmp/public.toml
```
Comment (jeffallen) [inline docs/setup.md]: tmp here too (this is where I first noticed it and wondered why it is tmp)

File: docs/setup.md [hunk 3]
```diff
@@ -100,7 +100,120 @@ in their associated documentation (chapter "Components").
...
+
+```bash
+# This file was created by the run_nodes.sh script
+bcadmin create ledger/conode/tmp/public.toml 
+# The output of the command offers you to export the BC variable. Copy/past the
+# last line into to your terminal
+export BC="path/to/folder/bc-BYZCOIN_ID.cfg"
+```
+
+It is also a good idead to save the `export BC=...` command in the
```
Comment (jeffallen) [inline docs/setup.md]:
```suggestion
It is also a good idea to save the `export BC=...` command in the
```

File: docs/setup.md [hunk 4]
```diff
@@ -100,7 +100,120 @@ in their associated documentation (chapter "Components").
...
+
+Launch the following commands or only those containing the rules you are
+interested in:
+
+```bash
+# Firstly, spot and note your identity (something like "ed25519:aef123...")
+bcadmin info
+# Save the identity into a variable
+id=IDENTITY
+# Note: do not run those commands into one single batch in order to let the
```
Comment (jeffallen) [inline docs/setup.md:200]: How about fixing `bcadmin darc rule` to take multiple rules and put them into one transaction? Sorry, I know it's kind of boring work, but...

File: domanager/app/controllers/session.go [hunk 5]
```diff
@@ -116,6 +117,17 @@ func sessionPost(w http.ResponseWriter, r *http.Request,
 			err.Error(), w, r, store)
 		return
 	}
+
+	// Check if the upload folder exist and create it if not
+	_, err = os.Stat("test")
```
Comment (jeffallen) [inline domanager/app/controllers/session.go]: test?

File: docs/setup.md (hunk 1 above)
Comment (nkcr) [reply docs/setup.md]: tmp is perhaps not the best choice, its an old habit when I use run_nodes.sh to quickly spin up a temporary cothority for development purposes. But still, I find it much more manageable to separate the cothority files and folders from the conode code. Its also easier to see and understand what the run_nodes.sh script do, and its also easier to exclude this single folder from git.

File: docs/setup.md (hunk 4 above)
Comment (nkcr) [reply docs/setup.md:200]: then we should take a configuration file as input and parse it, as we want (not in this particular case though) to specify N rules that can have each K different rules. Not impossible but requires some work in order to speed up a set up process that should happen only once.

This is not the first time we are limited with the edition of DARCs. There is also this story where we cannot edit a single rule but only replace it entirely. A "DARC editor" could be a funny semester project.

File: domanager/app/controllers/session.go (hunk 5 above)
Comment (nkcr) [reply domanager/app/controllers/session.go]: Ooops 0_o

**Pull request merged**